content/ADR-279-wearable-outline-compatibility.md

```diff
@@ -0,0 +1,102 @@
+---
+adr: 279
+date: 2025-01-13
+title: Wearable Outline Compatibility Metadata
+status: Draft
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - jhidalgo
+---
+
+## Context and Problem Statement
+
+The current avatar rendering system uses inverted face rendering for outlines in Medium/High quality settings. This creates conflicts with custom meshes that have manual outlines or inverted normals, resulting in incorrect rendering. Currently, there is no way to identify incompatible assets before they are rendered, leading to visual artifacts and inconsistent appearance.
```

Comment:
This sentence assumes there is only one renderer in use, which is not accurate as there are alternative clients as well. I would suggest we specify that we talk about `unity-explorer`

Reply:
@aixaCode thanks for the feedback Aga, sentence updated, let me know if it looks fine now so we're good to merge :)

Reply:
@aixaCode would you mind resolving the change request if it looks good now? :) so I can finally merge it

Reply:
@aixaCode :)